docs/tasks.md

```diff
@@ -0,0 +1,77 @@
+# Cumulus Tasks
```

Comment:
I like to include files like this in git because there are people (like me) that will go and read things in the repo rather than the built site, so it's nice to have it available.